common/build.gradle.kts

```diff
@@ -11,6 +11,11 @@ repositories {
 
 dependencies {
   testImplementation(kotlin("test"))
+  testImplementation("org.junit.jupiter:junit-jupiter:5.10.0")
```

Comment:
This is already present in https://github.com/TBD54566975/web5-kt/blob/main/build.gradle.kts#L46

Curious why it's needed here?